src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
+@SuppressWarnings("unchecked")
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int size;
+    private int capacity;
+    private int threshold;
```

Comment:
you can declare the threshold variable in initializeTable() method where you are initialized it. The same for capacity.

Reply:
I have the same issue, you can initialize it dynamically, but I don't think it is an error :) so I accepted your variant too :)